src/components/editor/metaform-editor-right-drawer.tsx

```diff
@@ -49,6 +51,28 @@ const MetaformEditorRightDrawer: React.FC<Props> = ({
     setPendingForm(updatedForm);
   };
 
+  /**
+   * Updates metaform field VisiblityIf
+   *
+   * @param newMetaformField new metaform field
+   */
+  const updateFormFieldVisiblityValues = (newMetaformField: MetaformField, eventValue: string) => {
+    setRequiredFieldBoolean(eventValue);
+    if (sectionIndex === undefined || fieldIndex === undefined) {
+      return;
+    }
+    const field = 2;
+    
+    if (field === undefined) {
+      return;
+    }
```

Comment:
What is this doing? Constant _field_ is not being used anywhere else so this looks kinda odd.

Reply:
Deleted it as it was useless.